locales/id.json

```diff
@@ -25,8 +25,16 @@
     "1 min read": "1 menit dibaca",
     "% min read": "% menit dibaca",
     "Article": "Artikel",
+    "Archive": "Arsip",
     "Author": "Penulis",
     "Topic": "Topik",
+    "Topics": "Topik-topik",
     "Share": "Bagikan",
-    "Search": "Pencarian"
+    "Search": "Pencarian",
+    "Search all posts": "Cari semua pos",
+    "Main": "Utama",
+    "Footer": "Footer",
```

Comment:
The English terms "footer" commonly used instead of the proper "Kaki dokumen" (lit: document foot). If any Indonesians would like to use the proper one, just notify me.